app/api/v2/users/views.py

```diff
@@ -0,0 +1,108 @@
+import re
+import datetime
+from flask import request, json
+from flask_restful import Resource, reqparse, inputs
+from flask_jwt_extended import create_access_token
+from werkzeug.security import generate_password_hash, check_password_hash
+
+from .models import UserModel
+
+class UserSignup(Resource):
+    def post(self):
+        parser = reqparse.RequestParser()
+        parser.add_argument('firstname', required=True, location="json", type=inputs.regex(r'^(?!\s*$).+'), help="First Name cannot be blank.")
```

Comment:
create a function to handle validation of blank of wrong format fields so that you don't repeat yourself in a separate file